context-free.md

```diff
@@ -2,21 +2,31 @@
 
 ##  Who is this programming language for?
 
+This programming language seems to be primarily for graphic designers (either professionally or casually) who have some experience writing code and thinking mathematically. I could also see mathematicians who focus on geometry using it to play around with an idea, since it easily represents fractal images that would be hard to represent manually with something like TIKz.
 
 ## What is easy to do in this language? Why is it easy?
 
+This language makes it very easy to represent very complicated and abstract shapes. For example, you can represent an "infinite" amount of shapes with just a few lines. Actually having to specify everything in the image would be incredibly hard, but Context Free allows you to recursively specify shapes with properties as the reccurence relation. 
 
 ## What is hard to do in this language? Why is it hard?
 
+There were a few issues I ran into with the language. 
+ - First, I found it was occasionally hard to figure out what coordinates my images were centered around. Since the image window changes to fit your image, sometimes its hard to tell where the origin is. 
+ - Additionally, the syntax error messages weren't very descriptive, so debugging in this language isn't easy.
```

Comment:
I think there are other more fundamentally difficult things to do, such as creating, say, "realistic" looking images like faces or nature. I think that you ran into valid difficulties, but they don't touch on actual downsides in the intention of the language. That is to say, the language intentionally doesn't support those types of images, as opposed to unintentionally being annoying to use.